lib/generators/declare_schema/migration/migrator.rb

```diff
@@ -477,7 +511,7 @@ def change_table(model, current_table_name)
         end
 
         def change_indexes(model, old_table_name, to_remove)
-          Migrator.disable_constraints and return [[], []]
+          Migrator.disable_indexing and return [[], []]
```

Comment:
Should we a global config option to disable indexing? It seems to follow an almost identical pattern to disable fk constraints.

Reply:
Yes, good idea!